.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,32 @@
+name: 'K3S image update check'
+on:
+  # Triggers at 3:00 am on the 15th of every month 
+  schedule:
+    - cron: '0 3 15 * *'
```

Comment:
why don't we just trigger this every day? in an action "if" we can easily abort the workflow when there is no new k3s version. that way we get quicker feedback on the image build than monthly

Reply:
Kubernetes releases every three months. Why would you want to check for a new release every day?

Reply:
makes sense!